src/main/java/duke/customClass/LogicHandler.java

```diff
@@ -0,0 +1,152 @@
+     */
+    public void deadline(String input, List<Task> list) {
+        try {
+            String taskAndDate = input.split(" ", 2)[1];
+            String task = taskAndDate.split(" /by ")[0];
+            String date = taskAndDate.split(" /by ")[1];
+            Deadline deadline = new Deadline(task, date);
+            list.add(deadline);
+            System.out.println("added: " + deadline);
+            System.out.println("Now you have " + list.size() + " tasks in the list.");
+        } catch (ArrayIndexOutOfBoundsException e) {
+            System.out.println("Oops, your deadline requires both a description and date.");
+        }
+    }
+
+    /**
+     * Add an Event task to the list of tasks.
+     * @param input string input of user.
+     * @param list list of tasks.
+     */
+    public void event(String input, List<Task> list) {
+        try {
+            String taskAndDate = input.split(" ", 2)[1];
+            String task = taskAndDate.split(" /at ")[0];
+            String date = taskAndDate.split(" /at ")[1];
+            Event event = new Event(task, date);
+            list.add(event);
+            System.out.println("added: " + event);
+            System.out.println("Now you have " + list.size() + " tasks in the list.");
+        } catch (ArrayIndexOutOfBoundsException e) {
+            System.out.println("Oops, your Event requires both a description and date");
+        }
+    }
+
+    /**
+     * Deletes a task in the list of tasks given a specified task index integer.
+     * @param input string input of user.
+     * @param list list of tasks.
+     */
+    public void delete(String input, List<Task> list) {
+        try {
+            int index = Integer.valueOf(input.split(" ", 2)[1]);
+            System.out.println("Noted. I've removed the task:\n" + list.get(index - 1) +
+                    "\nNow you have " + (list.size() - 1) + " tasks in the list.");
+            list.remove(index - 1);
+        } catch (ArrayIndexOutOfBoundsException e) {
+            System.out.println("Delete requires a number");
+        } catch (IndexOutOfBoundsException e) {
+            System.out.println(e);
+        } catch (NumberFormatException e) {
+            System.out.println(e);
+        }
+    }
+
+    public void find(String input, List<Task> list) {
+        try {
+            Find find = new Find();
+
+            String itemToFind = input.split(" ", 2)[1];
+            List<Task> newList = find.contains(itemToFind, list);
+
+            String temp = "";
+
+            for (int i = 0; i < newList.size(); i++) {
+                temp += String.format("%d. %s", i + 1, newList.get(i));
+                if (i != newList.size() - 1) {
+                    temp += "\n";
+                }
+            }
+            System.out.println(temp);
+        } catch (ArrayIndexOutOfBoundsException e) {
+            System.out.println("Oops, your find requires a string description to find a match.");
+        }
+    }
```

Comment:
Since this isn't a constructor, setter or getter, perhaps a header comment should be included here?